gluster-exporter/metric_brick.go

```diff
@@ -129,14 +129,15 @@ func diskUsage(path string) (disk DiskStatus, err error) {
 }
 
 func brickUtilization() {
-	volumes, err := glusterutils.VolumeInfo(&glusterConfig)
+	gluster := glusterutils.MakeGluster(&glusterConfig)
```

Comment:
This can be moved to `main.go`. Make `gluster` as global variable similar to `glusterConfig`

Reply:
the actual config check is done in main() function in main.go. if i make gluster = MakeGluster(&config) before we parse the global conf, doesnt it always return gd1 instance?.

Reply:
it should be after parsing the config and before starting the metrics collectors

Reply:
I just declared gluster var as global and assigned it in main().